features/step_definitions/steps.rb

```diff
@@ -159,7 +162,9 @@
 end
 
 And(/^I click "([^"]*)"$/) do |value|
-  find_button(value).trigger('click')
+  within 'main' do
```

Comment:
I had some issues with Poltergeist being able to interact with the links/buttons using Capybara so I created 2 step defs. A bit hacky I know